src/python/pants/backend/python/util_rules/dists.py

```diff
@@ -157,7 +157,15 @@ class DistBuildResult:
 wheel_config_settings = {wheel_config_settings_str}
 sdist_config_settings = {sdist_config_settings_str}
 
-os.makedirs(dist_dir, exist_ok=True)
+# Python 2.7 doesn't have the exist_ok arg on os.makedirs(), so we have to emulate it.
+def safe_mkdir(path):
+    if not os.path.exists(path):
```

Comment:
This seems a lot easier to know at a glance is correct: https://github.com/pantsbuild/pex/blob/2693db7c1ec6e1c810126820f250db206391969a/pex/common.py#L271-L275

Bonus, it avoids check and act races. Probably not a concern here though.

Reply:
Oh yeah, that is nicer and simpler. Will do that.